src/ProjectTemplates/Microsoft.McpServer.ProjectTemplates/templates/McpServer-CSharp/remote/README.md

```diff
@@ -0,0 +1,58 @@
+# MCP Server
+
+This README was created using the C# MCP server project template.
+It demonstrates how you can easily create an MCP server using C# and run it as an ASP.NET Core web application.
+
+#### ---#if (SelfContained)
+The MCP server is built as a self-contained application and does not require the .NET runtime to be installed on the target machine.
+However, since it is self-contained, it must be built for each target platform separately.
+By default, the template is configured to build for:
+* `win-x64`
+* `win-arm64`
+* `osx-arm64`
+* `linux-x64`
+* `linux-arm64`
+* `linux-musl-x64`
+
+If your require more platforms to be supported, update the list of runtime identifiers in the project's `<RuntimeIdentifiers />` element.
+#### ---#else
+The MCP server is built as a framework-dependent application and requires the ASP.NET Core runtime to be installed on the target machine.
+The application is configured to roll-forward to the next highest major version of the runtime if one is available on the target machine.
+If an applicable .NET runtime is not available, the MCP server will not start.
+Consider building the MCP server as a self-contained application if you want to avoid this dependency.
+#### ---#endif
+
+Please note that this template is currently in an early preview stage. If you have feedback, please take a [brief survey](http://aka.ms/dotnet-mcp-template-survey).
+
+## Developing locally
+
+To test this MCP server from source code (locally), you can configure your IDE to connect to the server using localhost.
```

Comment:
Maybe we should add a blurb here about creating an mcp.json file with the below information to be more specific?